pkg/ui/src/views/app/components/layoutSidebar/index.tsx

```diff
@@ -70,7 +71,13 @@ function LoginIndicator({ loginState, handleLogout }: LoginIndicatorProps) {
   }
 
   if (!loginState.loginEnabled()) {
-    return (<div className="login-indicator">Insecure mode</div>);
+    return (
```

Comment:
I don't see this indicator when I start a cluster in insecure mode. Just blank space above the cockroach icon.

Reply:
Are you sure you had the environment variable set?  I am seeing it...